README.md

```diff
@@ -8,8 +8,12 @@ yarn add @unleash/proxy-client-react unleash-proxy-client
 
 # How to use
 
+This library uses the core [unleash-proxy-client](https://github.com/Unleash/unleash-proxy-client-js) JS client as a base.
+
 ## Initialize the client
 
+*NOTE*: [unleash-proxy](https://github.com/Unleash/unleash-proxy) is in maintenance moded. It is recommend to use the [Frontend API](https://docs.getunleash.io/reference/front-end-api) or [unleash-edge](https://github.com/Unleash/unleash-edge) instead
```

Comment:
```suggestion
*NOTE*: [unleash-proxy](https://github.com/Unleash/unleash-proxy) is in maintenance mode. It is recommend to use the [Frontend API](https://docs.getunleash.io/reference/front-end-api) or [unleash-edge](https://github.com/Unleash/unleash-edge) instead.
```

Reply:
We can do that, however maybe it will confuse some people to think that proxy-client-react is in maintainence mode?

Reply:
So if proxy-client is in maintenance mode, and react-proxy-client depends on proxy-client, wouldn't it make sense to have them both in maintenance mode? maybe not necessarily?

Reply:
@melindafekete 

Yeah.. This is confusing. Alle our frontend SDKs were named "Proxy-X-X" once upon a time because they were meant to be used with the Proxy. Then we developed Edge, which is the proxy successor and all of our SDKs work with Edge as well. So even though we put proxy in maintenance mode all of our consumer SDKs will still be maintained.

We should change the name of these SDKs.